src/containers/LoginForm.tsx

```diff
@@ -166,8 +146,8 @@ const LoginForm = () => {
     <>
       <ModalUniversal
         button={t("login.loginButton")}
-        title=""
-        content={LoginFormContainer}
+        title={currentUserId ? "close me" : ""}
```

Comment:
use i18n for title

Reply:
This is a temporary change in the display of the modal window.